src/geocoder.js

```diff
@@ -28,14 +37,18 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
```

Comment:
From the [Geocoding Authentication Guide](https://developers.google.com/maps/documentation/static-maps/get-api-key):
> The Google Maps Geocoding API will only work with a Server key
Maybe we shouldn't even support unauthenticated mode?